### PR TITLE
CI: Add workflow to test install via conda-forge.

### PR DESCRIPTION
**Pull request by rossbar**
Add a GitHub workflow to test installing `pygraphviz` with `conda` via the conda-forge channel.

### CODE REVIEW

**Comment by rossbar** [comment]
This PR adds a scheduled job to the CI to test installing/running pygraphviz with conda via the conda-forge channel on windows, ubuntu, and macos. The results of the scheduled build are then [referenced in the install documentation with status badges](https://351-11846030-gh.circle-artifacts.com/0/doc/build/dist/install.html#install). Note that currently the badges say "no status" - my hypothesis is that this is because the scheduled jobs won't actually start running until the workflow is merged.

The idea is to:
 1. have an automated process that monitors the status of installing pygraphviz with conda, which has historically been problematic (and still is for the default channels), and
 2. make the latest info on the status of installing via conda visible to users.

One downside of the badge approach is that it does not appear possible to create status badges for individual jobs *within* a single workflow. As a workaround I've added three separate workflow files, one for each platform. I've also opened a feature request at GitHub, so if it is/becomes possible to create job-level badges, the three new workflows can be consolidated.

**Comment by rossbar** [comment]
A gentle ping on this PR. We've recently gotten some issues indicating potential problems with the recommended `conda-forge` installation steps. This PR would setup cron jobs to verify (and notify users via badges) that installing pygraphviz via the conda-forge channel is working as expected on windows, mac, and linux.

File: .github/workflows/conda-macos.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: conda-forge-macos
+on:
+  schedule:
+    # Run this workflow once a day
+    - cron: "0 0 * * *"
```
Comment (MridulS) [inline .github/workflows/conda-macos.yml:5]: Just a little nit pick, but can we schedule to weekly/biweekly instead?
Let's save some compute time and I don't think there are that many changes
happening in the build pipeline of pygraphviz?

@rossbar What do you think?

Comment (rossbar) [reply .github/workflows/conda-macos.yml]: Sounds good to me, I switched the cron scheduling to weekly for the conda-forge jobs.

**Review by MridulS — APPROVED** [review]

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**